### PR TITLE
[publication] help dialog info

### LINKED ISSUE #4593: [Publications] Missing drop-down Help text
No drop-down help text is available in the 21.0.0 release branch for the new Publications module when a user clicks on the question mark in the top menu bar.

### PR DESCRIPTION
**Pull request by maltheism**
### Brief summary of changes

Added Readme.md to be the help info for the module because it describes the module. 

### This resolves issue...

- [ ] Redmine? #

- [x] Github? Resolves #4593

### To test this change...

- [x] go to publication module, click the ?

### CODE REVIEW

File: modules/publication/help/publication.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- `publication_propose`: grants the user access to proposing their research projects
+and uploading relevant publication media.
+- `publication_approve`: grants the user access to reviewing (accepting or rejecting)
+project proposals.
+
+## Configurations
+
+- `publication_uploads`: This configuration determines the directory where file uploads
+ get stored. By default, this directory will be `/data/publication_uploads/` but
+  it is not automatically created.
```
Comment (johnsaigle) [inline modules/publication/help/publication.md]: There is also a `publication_deletions` setting which should be documented.

File: modules/publication/help/publication.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Publication
+
+The publication module allows users to propose research projects based on LORIS 
+data, upload publication media, and allows study PIs to review the project 
+proposals.
+
+## Intended Users
```
Comment (driusan) [inline modules/publication/help/publication.md]: The help directory is for user-facing help that gets loaded when clicking the "?" icon on the front end which should explain how to use the module. This seems to be specifications for the README.md, not help text.

File: modules/publication/help/publication.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+and uploading relevant publication media.
+- `publication_approve`: grants the user access to reviewing (accepting or rejecting)
+project proposals.
+
+## Configurations
+
+- `publication_uploads`: This configuration determines the directory where file uploads
+ get stored. By default, this directory will be `/data/publication_uploads/` but
+  it is not automatically created.
+- `publication_delections`: Only users that are affiliated with the proposal can delete.
```
Comment (johnsaigle) [inline modules/publication/help/publication.md]:
```suggestion
- `publication_deletions`: Only users that are affiliated with the proposal can delete.
```

**Comment by maltheism** [comment]
@johnsaigle @driusan I modified the README.md content to be more helpful in context of users needing help in how to use the module. Please let me know how it can be improved.

File: modules/publication/help/publication.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Publication
+
+Publication is intended for users to propose specific research projects
+and for PIs to review them. When proposing a project, PIs may also specify 
+other users to help with editing and maintaining their project proposals.
+
+
+Navigate to the "__Propose a Project__" tab for proposing research projects based on the data required. You may upload publication media, and allows study PIs to review the project
```
Comment (driusan) [inline modules/publication/help/publication.md]: The second part of the second sentence "and allows study PIs" isn't grammatically correct. Maybe something like "which may then be reviewed by study PIs" would be better.

File: modules/publication/help/publication.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+Publication is intended for users to propose specific research projects
+and for PIs to review them. When proposing a project, PIs may also specify 
+other users to help with editing and maintaining their project proposals.
+
+
+Navigate to the "__Propose a Project__" tab for proposing research projects based on the data required. You may upload publication media, and allows study PIs to review the project 
+proposals.
+
+Specify which specific variables of
```
Comment (driusan) [inline modules/publication/help/publication.md]:
```suggestion
You should specify which specific variables of interest are relevant to your research so
that the reviewers can determine if they should grant access to the data.
```

The suggested change is to replace these 3 lines, but I can't do multiline suggestions. (I don't think the line about module scope is relevant to the users. They might want to know how they'll get the data, but we don't have any control over that so I can't think of any way to write general purpose documentation..)

Comment (maltheism) [reply modules/publication/help/publication.md]: @driusan how about now?

**Pull request merged**